apps/explorer/lib/explorer/chain.ex

```diff
@@ -3654,6 +3654,32 @@ defmodule Explorer.Chain do
     end
   end
 
+  def query_leaderboard do
+    result =
+      SQL.query(Repo, """
+        SELECT competitors.address, b.name, SUM(rate*value+fetched_coin_balance+celo_account.locked_gold)*multiplier AS score
+        FROM addresses, exchange_rates, competitors, claims, celo_account, celo_account as b,
+          (SELECT claims.claimed_address AS address, COALESCE(SUM(value),0) AS value
+          FROM address_current_token_balances, claims
+          WHERE address_hash=claims.claimed_address
+          AND token_contract_address_hash='\\x88f24de331525cf6cfd7455eb96a9e4d49b7f292'
```

Comment:
What are these constants?

Reply:
It's the address for stable token. I guess this query will be thrown away after the stake off so it doesn't matter much that it's hard coded.

Reply:
Please add a comment for what this query does, or at least a link to to documentation